docs/release_notes/v1.10.7.md

```diff
@@ -17,3 +17,29 @@ The latest versions (1.38.X) of the shopify/sarama kafka library introduced this
 ### Solution
 
 Downgrading the shopify/sarama library to release 1.37.2 resolves this problem. An issue with the maintainers of the sarama library has been created for further investigation.
+
+## Fixed an issue with HTTP URL routing in Dapr sidecar
+
+### Problem
+
+When using the Dapr sidecar HTTP API, some crafted HTTP URLs where not being routed to
+the correct handler, for example, when prefixing the URL path with duplicated
+slashes. i.e. `//v1.0/actors/DemoActor/1234/method/SetReminder`.
+
+### Impact
+
+Requests which did not have perfect paths according to the API may be routed to
+the wrong API handler, and typically return some kind of error.
+
+### Root cause
+
+The root HTTP request handler was not normalizing the request's HTTP path before
+passing it to the router to evaluate which handler to execute the request. Note
+that, the path _was_ normalized by the time it reached the underlying API
+handler.
+
+### Solution
+
+Use the `URI().SetQueryString` and `URI().SetPath` functions, rather than the
```

Comment:
Keep it generic, since this won't mean much for someone who's not a Dapr core developer.

> We changed the router used by the Dapr HTTP server to "normalize" HTTP paths before matching the route.